sample_google_maps_data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "markers": [
+    {
+      "name": "Rixos The Palm Dubai",
+      "position": [25.1212, 55.1535],
```

Comment:
One of my students pointed out that `position` here doesn't match `location` below. As long as we're taking ownership, let's change this.